Framework/API/test/DetectorInfoTest.h

```diff
@@ -393,8 +393,8 @@ class DetectorInfoTest : public CxxTest::TestSuite {
     size_t numberOfBins = 1;
     workspace.initialize(numberOfHistograms, numberOfBins + 1, numberOfBins);
     for (int32_t i = 0; i < numberOfHistograms; ++i)
-      workspace.getSpectrum(i)
-          .setSpectrumNo(static_cast<int32_t>(numberOfHistograms) - i);
+      workspace.getSpectrum(i).setSpectrumNo(
```

Comment:
I think this should be done through `workspace.mutableIndexInfo()`

Reply:
I didn't actually modify this file - I think clang format changed some stuff... Should I still make this change?

Reply:
I'll leave that up to you

Reply:
I'll try add the changes